Fitness/Views/Cells/FacilityHoursCell.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  FacilityHoursCell.swift
+//  Fitness
+//
+//  Created by Ji Hwan Seung on 4/24/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import SnapKit
+import UIKit
+
+class FacilityHoursCell: UITableViewCell {
+    
+    // MARK: - INITIALIZATION
+    static let identifier = Identifiers.facilityHoursCell
+    var dayLabel: UILabel!
+    var hoursLabel: UILabel!
+    var hoursScrollView: UIScrollView!
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        dayLabel = UILabel()
+        dayLabel.font = UIFont._12MontserratBold
+        dayLabel.textColor = .fitnessBlack
+        dayLabel.textAlignment = .left
+        dayLabel.sizeToFit()
+        dayLabel.text = "MON"
+        contentView.addSubview(dayLabel)
+        
+        hoursScrollView = UIScrollView()
+        hoursScrollView.contentSize.height = 1.0
+        hoursScrollView.alwaysBounceVertical = false
+        hoursScrollView.showsVerticalScrollIndicator = false
+        contentView.addSubview(hoursScrollView)
+        
+        hoursLabel = UILabel()
+        hoursLabel.font = ._16MontserratLight
+        hoursLabel.textColor = .fitnessBlack
+        hoursLabel.textAlignment = .left
+        hoursLabel.sizeToFit()
+        hoursLabel.text = "6: 00 AM - 9: 00 PM"
+        hoursScrollView.addSubview(hoursLabel)
+        
+        setupConstraints()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - CONSTRAINTS
+    func setupConstraints() {
+        dayLabel.snp.updateConstraints {make in
+            make.leading.equalToSuperview().offset(22)
```

Comment:
Can we bring all these constants up to the top of the function according to bible?